libs/utils/src/lib/serializer.service.ts

```diff
@@ -10,12 +10,13 @@ export class Serializer {
       ...f,
       important:
         f.positional ||
+        f.required ||
```

Comment:
Some fields are 'required' by Angular Console while being buried in the 'Optional Fields' panel. 

I added this because I think this was an oversight - but if it was intentional than maybe 'Optional' should be renamed in the UI so as not to be misleading.

Reply:
Nope, you nailed it. That was a bad over sight. If you wouldn't mind could you add a regression test to make sure it doesn't get into a bad state again?